pyodbc.pyi,sha256=JWNpdgmrxhNioTjW6OoAEcXELcGZobcMYzKtYlcJRuY,32320
pyodbc.cpython-310-aarch64-linux-gnu.so,sha256=G4sOJR57n4UskmhrIxlC_NauyRDWn7ES3nw9Vvh4Iy4,1055056
pyodbc-4.0.38.dist-info/WHEEL,sha256=5FPtA1B6n-DIbcPbXnHVzp-as0KLsaNL-sSVkjvwO_0,154
pyodbc-4.0.38.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-4.0.38.dist-info/RECORD,,
pyodbc-4.0.38.dist-info/METADATA,sha256=EjZZ8cmsdLrMbWGabwt6HcQGnUcZ89VyEbCiebo_njY,3160
pyodbc-4.0.38.dist-info/top_level.txt,sha256=Rb69vS-SNN_Bfh99vX7eCcvEAJoyzowG3u4R7W36Flw,8
